terraform/application/application.tf

```diff
@@ -70,4 +70,5 @@ module "worker_application" {
   replicas                   = var.worker_replicas
   docker_image               = var.docker_image
   enable_logit               = true
+  enable_gcp_wif             = true
```

Comment:
this should be
enable_gcp_wif = var.enable_dfe_analytics_federated_auth
otherwise it enables on all environments